### PR TITLE
Updated get_cpds method: ensured consistent return type and improved error handling 

### LINKED ISSUE #933: Correct docstring of get_cpds of BayesianModels
The docstring of `get_cpds` in `BayesianModel` doesn't specify that it returns a TabularCPD object when `node` is not `None. Specify this in the docstrings and also need to improve doctests.

### PR DESCRIPTION
**Pull request by B3CODER**
…error handling

### Your checklist for this pull request
Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [ x ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [ x ] Make sure you are making a pull request against the **dev branch** (left side). Also you should start *your branch* off *our dev*.
- [ x ] Check the commit's or even all commits' message styles matches our requested structure.

### Issue number(s) that this pull request fixes
- Fixes #933 

### List of changes to the codebase in this pull request
- Updated the 'get_cpds' method in 'BayesianNetwork.py' to provide consistent output for both single node CPDs and all CPDs.
- Added error handling for cases where a node is not present in the graph.

### CODE REVIEW

File: model.bif [hunk 1]
```diff
@@ -0,0 +1,32 @@
+network unknown {
```
Comment (ankurankan) [inline model.bif]: I am not sure if you have accidentally committed this file. Could you please remove it?

Comment (B3CODER) [reply model.bif]: where to check ? i didn't do this changes but i can also not find

Comment (ankurankan) [reply model.bif]: @B3CODER You can remove it using:
```
git rm model.bif
git commit -m "Removes model.bif"
```

**Comment by ankurankan** [comment]
@B3CODER Looks good. Thanks for the PR.

**Pull request merged**